package.json

```diff
@@ -48,21 +48,21 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "babel-cli": "^6.3.17",
-    "babel-eslint": "^5.0.0-beta6",
-    "babel-preset-es2015": "^6.3.13",
-    "babel-preset-react": "^6.3.13",
-    "babel-register": "^6.3.13",
-    "chai": "^3.4.1",
-    "enzyme": "^1.2.0",
-    "jsdom": "^6.1.0",
-    "mocha": "^2.3.4",
-    "react": "^0.14.0",
-    "react-addons-test-utils": "^0.14.3",
-    "react-dom": "^0.14.0",
+    "babel-cli": "6.5.1",
```

Comment:
Happy to prepend them all with `^`, this was a 'best practice' I follow on application code. I didn't think twice in this case since these are dev dependencies